prime-router/docs/design/proposals/0023-condition-to-code-mapping/0023-condition-to-code-mapping.md

```diff
@@ -0,0 +1,383 @@
+
+```yaml
+- name: "unmapped-message-queue"
+  description: "All messages that contain unmapped observations"
+  jurisdiction: "FEDERAL"
+  filters: "Bundle.entry.resource.ofType(Observation).resource.meta.tag.code.exists().not()"
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+      - name: "unmapped-message-queue"
+        organizationName: "
+      topic: "full-elr"
+        customerStatus: "active"
+        translation:
+            schemaName: "metadata/hl7_mapping/receivers/STLTs/NY/NY-receiver-transform"
+            useBatchHeaders: false
+            type: "FHIR"
+        jurisdictionalFilter: []
+        qualityFilter: []
+        routingFilter: []
+        reverseTheQualityFilter: false
+        conditionFilter: []
+        deidentify: false
+        deidentifiedValue: ""
+        timing:
+            operation: MERGE
+            numberPerDay: 1440 # Every minute
+            initialTime: 00:00
+        transport:
+            type: SFTP
+            host: BLANK
+            port: BLANK
+            filePath: BLANK
+            credentialName: BLANK
+        externalName: null
+        timeZone: null
+        dateTimeFormat: "OFFSET"
+```
+This will gather all unmapped messages into a single queue and also fire a "last mile failure" alert that will automatically ping the engagement slack channel when a message is added to the queue.
+
+## Automated Mapping updates
+
+Since the bulk of the mapping table will be made up of values taken from CSTE codesets that are stored in the Value Set Authority Center (VSAC). We can make use of the VSAC's [FHIR Terminology Service](https://www.nlm.nih.gov/vsac/support/usingvsac/vsacfhirapi.html) to query and update the table. Using the "Member OID" and "Version" columns from the table we can create a utility to query each Member OID for which the "Value Source" column is "RCTC" to see if we have the latest version and if not to update to the latest version. This should be a CLI utility similar to update-livd-table command that can be run on-demand to update the table.
+
+The FHIR Terminology Service for VSAC Resources is a RESTful API service for accessing the VSAC value sets and supported code systems.
+
+Example parameters: </br>
+environment - name of table to compare input to </br>
+silent - do not generate diff or ask for confirmation
+activate - activate the table upon creation
+key - apikey to access VSAC FHIR terminology service
+OID - OID of valueset to update (default to all OID in table if blank)
+
+Example CLI command for update tool:
+
+./prime condition-mapping-update --activate --silent --environment staging --key <apikey>
+
+## Sender Compendium Comparison Utility
```

Comment:
Should update this section to make it clear that we are checking code and codeSystem for equivalency and ignoring description, per our discussion